service/src/intTest/java/uk/nhs/adaptors/gp2gp/gpc/GetGpcDocumentComponentTest.java

```diff
@@ -37,6 +37,7 @@
 @DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_EACH_TEST_METHOD)
 public class GetGpcDocumentComponentTest extends BaseTaskTest {
     private static final String DOCUMENT_NAME = EhrStatusConstants.DOCUMENT_ID + ".json";
+    private static final String DOCUMENT_URL = "https://orange.testlab.nhs.uk/B82617/STU3/1/gpconnect/documents/fhir/Binary/";
```

Comment:
This should not be hardcoded. Build it up from properties defined in intTest/resources/application.yml

Our integration tests should be using wiremock by default. It doesn't look like that happened in the last sprint. I'll enter a new story for it.